Cannot set 'topic.offset.map' parameter

**Description:**
Setting  'topic.offset.map' parameter gives Siddhi Parse Error on siddhi app deployment. 

**Affected Product Version:**
Stream Processor 4.3.0 and Stream Processor 4.4.0

**OS, DB, other environment details and versions:** 
N/A   

**Steps to reproduce:**
Deploy following Siddhi app and the following error log will be printed on the console, during Siddhi app deployment time. 

```
@App:name("HelloKafka")

@source(type='kafka',
        topic.list='production',
        threading.option='single.thread',
        group.id="group3",
        bootstrap.servers='localhost:9092',
        topic.offset.map='production=2',
        @map(type='json'))
define stream SweetProductionStream (name string, amount double);

@sink(type='log')
define stream OutputStream (name string, amount double);

from SweetProductionStream
select *
insert into OutputStream;
```
```
[2019-07-05 16:06:59,608] ERROR {org.wso2.carbon.stream.processor.core.internal.StreamProcessorDeployer} - org.wso2.carbon.stream.processor.core.internal.exception.SiddhiAppConfigurationException: Exception occurred when retrieving Siddhi App Name org.wso2.carbon.stream.processor.core.internal.exception.SiddhiAppDeploymentException: org.wso2.carbon.stream.processor.core.internal.exception.SiddhiAppConfigurationException: Exception occurred when retrieving Siddhi App Name
	at org.wso2.carbon.stream.processor.core.internal.StreamProcessorDeployer.deploySiddhiQLFile(StreamProcessorDeployer.java:106)
	at org.wso2.carbon.stream.processor.core.internal.StreamProcessorDeployer.deploy(StreamProcessorDeployer.java:330)
	at org.wso2.carbon.deployment.engine.internal.DeploymentEngine.lambda$deployArtifacts$0(DeploymentEngine.java:291)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.wso2.carbon.deployment.engine.internal.DeploymentEngine.deployArtifacts(DeploymentEngine.java:282)
	at org.wso2.carbon.deployment.engine.internal.RepositoryScanner.sweep(RepositoryScanner.java:112)
	at org.wso2.carbon.deployment.engine.internal.RepositoryScanner.scan(RepositoryScanner.java:68)
	at org.wso2.carbon.deployment.engine.internal.DeploymentEngine.start(DeploymentEngine.java:121)
	at org.wso2.carbon.deployment.engine.internal.DeploymentEngineListenerComponent.onAllRequiredCapabilitiesAvailable(DeploymentEngineListenerComponent.java:216)
	at org.wso2.carbon.kernel.internal.startupresolver.StartupComponentManager.lambda$notifySatisfiableComponents$7(StartupComponentManager.java:266)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.wso2.carbon.kernel.internal.startupresolver.StartupComponentManager.notifySatisfiableComponents(StartupComponentManager.java:252)
	at org.wso2.carbon.kernel.internal.startupresolver.StartupOrderResolver$1.run(StartupOrderResolver.java:204)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.wso2.carbon.stream.processor.core.internal.exception.SiddhiAppConfigurationException: Exception occurred when retrieving Siddhi App Name
	at org.wso2.carbon.stream.processor.core.internal.StreamProcessorService.getSiddhiAppName(StreamProcessorService.java:338)
	at org.wso2.carbon.stream.processor.core.internal.StreamProcessorDeployer.deploySiddhiQLFile(StreamProcessorDeployer.java:92)
	... 14 more
Caused by: org.wso2.siddhi.query.compiler.exception.SiddhiParserException: Error between @ Line: 11. Position: 0 and @ Line: 11. Position: 14. Syntax error in SiddhiQL, no viable alternative at input '@source(type=kafka,\n        topic.list=kafka_topic,\n        threading.option=single.thread,\n        group.id=group,\n        bootstrap.servers=localhost:9092,\n        topic.offset'.
	at org.wso2.siddhi.query.compiler.internal.SiddhiErrorListener.syntaxError(SiddhiErrorListener.java:36)
	at org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:65)
	at org.antlr.v4.runtime.Parser.notifyErrorListeners(Parser.java:566)
	at org.antlr.v4.runtime.DefaultErrorStrategy.reportNoViableAlternative(DefaultErrorStrategy.java:308)
	at org.antlr.v4.runtime.DefaultErrorStrategy.reportError(DefaultErrorStrategy.java:145)
	at org.wso2.siddhi.query.compiler.SiddhiQLParser.siddhi_app(SiddhiQLParser.java:474)
	at org.wso2.siddhi.query.compiler.SiddhiQLParser.parse(SiddhiQLParser.java:178)
	at org.wso2.siddhi.query.compiler.SiddhiCompiler.parse(SiddhiCompiler.java:69)
	at org.wso2.carbon.stream.processor.core.internal.StreamProcessorService.getSiddhiAppName(StreamProcessorService.java:325)
	... 15 more
```

--- a/component/src/main/java/io/siddhi/extension/io/kafka/source/KafkaSource.java
+++ b/component/src/main/java/io/siddhi/extension/io/kafka/source/KafkaSource.java
@@ -102,7 +102,7 @@ import java.util.concurrent.ScheduledExecutorService;
                         type = {DataType.BOOL},
                         optional = true,
                         defaultValue = "false"),
-                @Parameter(name = "topic.offset.map",
+                @Parameter(name = "topic.offsets.map",
                         description = "This parameter specifies reading offsets for each topic and partition. " +
                                 "The value for this parameter is specified in the following format: \n " +
                                 "`<topic>=<offset>,<topic>=<offset>,`\n " +
@@ -171,7 +171,7 @@ public class KafkaSource extends Source<KafkaSource.KafkaSourceState> implements
     public static final String ADAPTOR_SUBSCRIBER_ZOOKEEPER_CONNECT_SERVERS = "bootstrap.servers";
     public static final String ADAPTOR_SUBSCRIBER_PARTITION_NO_LIST = "partition.no.list";
     public static final String ADAPTOR_OPTIONAL_CONFIGURATION_PROPERTIES = "optional.configuration";
-    public static final String TOPIC_OFFSET_MAP = "topic.offset.map";
+    public static final String TOPIC_OFFSET_MAP = "topic.offsets.map";
     public static final String THREADING_OPTION = "threading.option";
     public static final String SEQ_ENABLED = "seq.enabled";
     public static final String HEADER_SEPARATOR = ",";

